sdk/storage/azure-storage-files-datalake/src/datalake_lease_client.cpp

```diff
@@ -0,0 +1,11 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// SPDX-License-Identifier: MIT
+
+#include "azure/storage/files/datalake/datalake_lease_client.hpp"
+
+namespace Azure { namespace Storage { namespace Files { namespace DataLake {
+
+  const std::chrono::seconds DataLakeLeaseClient::InfiniteLeaseDuration{
```

Comment:
I don't know if this works. Google `static initialization order fiasco`

Reply:
You are actually right, didn't realize that.